src/ui/osx/TogglDesktop/LoginViewController.m

```diff
@@ -213,24 +193,29 @@ - (void)changeTabView:(TabViewType)type
 	switch (type)
 	{
 		case TabViewTypeLogin :
-			// Update nextkeyView
-			[self.password setNextKeyView:self.loginButton];
-			[self.loginGooglBtn setNextKeyView:self.email];
+            self.appleGoogleGroupViewTop.constant = 86.0;
+            self.containerViewHeight.constant = 440.0;
```

Comment:
this looks hacky, is there a way to not hardcode constants in the controller code?

Reply:
It would be nice if layout constants are only in .xib, but it's fine to keep it in the controller code if it's too difficult to do otherwise 👍